sources/docker-compose.yml

```diff
@@ -7,8 +7,8 @@ services:
     image: api-${PROJECT_NAME}:${BUILD_REF}-${BUILD_ID}
     container_name: api-${PROJECT_NAME}-${BUILD_REF}-${BUILD_ID}
     build:
-      context: ./packages/api
-      dockerfile: Dockerfile.dev
+      context: ./packages/backend
+      dockerfile: ./apps/api/Dockerfile.dev
```

Comment:
while declaring DOCKER_FILE_PATH, I thought we will use that property here. Am I missing something?

Reply:
`DOCKER_FILE_PATH` is a variable used in the Openshift files only and it is not related to the `docker-compose.yml`.